source/index.js

```diff
@@ -1,2 +1,7 @@
-require( "./angular-date-picker" );
-require( "./angular-time-picker" );
+"use strict";
+
+angular.module( "vokal.datePicker", [] )
+.directive( "datePicker", require( "./directives/date-picker" ) );
+
+angular.module( "vokal.timePicker", [] )
+.directive( "timePicker", require( "./directives/time-picker" ) );
```

Comment:
I'll split the definitions to separate files but keep this index file to `require` both.